dev/integration_tests/named_isolates/lib/main.dart

```diff
@@ -0,0 +1,19 @@
+import 'dart:ui' as ui;
+import 'package:flutter/material.dart';
+
+// named_isolates_test depends on these values.
+const String _kFirstIsolateName = 'first isolate name';
+const String _kSecondIsolateName = 'second isolate name';
+
+void first() {
+  main(_kFirstIsolateName);
+}
+
+void second() {
+  main(_kSecondIsolateName);
+}
+
+void main(String name) {
```

Comment:
This confused me for a minute. Using the name `main` like this steps on the convention of the normal `main` functions that either don't take any parameters or take `List<String>`. They are also usually entrypoints by themselves.